frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -89,6 +92,17 @@ const Map: React.FC<MapProps> = ({
   });
   const [baseLayers, setBaseLayers] = useState<BaseLayer[]>([]);
   const [activeBasemap, setActiveBasemap] = useState<BaseLayer | null>(null);
+  const smallScreen = useMediaQuery({ maxWidth: 1800 });
+
+  useEffect(() => {
```

Comment:
@devinleighsmith this is the code that trigger the fetch if there are no results in the current bounds

Reply:
Cool thanks, I get it now.